### DISCUSSION TITLE
help,is it possible to set the horizontal axis ticks?

### DISCUSSION THREAD

**Discussion by Siegward2000** [bokeh/bokeh#13695] [General]
How to adjust the horizontal axis when drawing a bar chart? Is it possible to set the horizontal axis ticks, such as displaying only 100, 200, etc. when the horizontal axis ranges from 1 to 1000? Thanks for your help.

**Comment by bryevdv** [comment]
There is information about specifying fixed tick locations in the docs:

https://docs.bokeh.org/en/latest/docs/user_guide/styling/plots.html#tick-locations

FYI GitHub Discussions are intended for development discussions of the project itself. For user support quesions, please use the community Discourse: 

https://discourse.bokeh.org
